### ISSUE TITLE
Playable Twitter Cards

### ISSUE THREAD

**Issue by povrazor** [ludumdare/ludumdare#79]
Reference is here:

https://github.com/twitterdev/cards-player-samples

Things to know:

* a Twitter Player card is either a preview (image) or played (iframe)

This means there's no way to do the 'deep integration' I was hoping to (i.e. bookmark or <3 Ludum Dare games) without a browser plugin.

**Comment by povrazor** [comment]
https://twitter.com/mikekasprzak/status/721422398491271168

![largeimagecard](https://cloud.githubusercontent.com/assets/1570313/14583533/956f0aae-03f2-11e6-9bd3-6c58e546cd68.png)

Alternatively, if we want to convey information, it can be done via the large image card.

Data can be either included in the description text, or if I'm feeling fancy, by pasting it on the image.

FWIW ImageMagick can be told what fonts to use on the command line.

`-font path/to/font/file.ttf`

**Comment by povrazor** [comment]
Check if Twitter Card meta tags can be set in JavaScript, or if they must be set live.

**Comment by povrazor** [comment]
http://ogp.me

OpenGraph is essentially the same thing, used by Facebook and Google.

According to this (http://stackoverflow.com/a/34522558/5678759) Google does execute JavaScript, but Facebook does not. In practice, I can more easily get JS meta tag emitting working.

Twitter has some compatibility with OpenGraph, so you can emit less meta tags if desired.

https://dev.twitter.com/cards/getting-started#opengraph

**Comment by povrazor** [comment]
No, Twitter Cards will not work as pure JS.

**Comment by blinry** [comment]
I really miss the Twitter Cards, compared to the old site, and I think it's worth working on bringing them back. Would the site architecture in general allow adding the meta tags server-side?

**Comment by povrazor** [comment]
Yes it is doable, but it's going to take some work. I'd say this is about a medium priority. i.e. not critical, but something we want sooner than later.
